### ISSUE TITLE
Unable to get working in Windows 10 64-bit with current method.

### ISSUE THREAD

**Issue by SpecialActivities** [CrazyRedMachine/Prodikeys64#3]
Is there a way to get the device working on Windows 10, 64-bit? i'd love to know! Current method didnt work sadly!

**Comment by CrazyRedMachine** [comment]
hi, please check this conversation, it contains all relevant information : https://github.com/CrazyRedMachine/Prodikeys64/issues/2

(tl;dr if you install the driver for both interfaces it will make the midi keyboard work but you'll lose the regular keyboard keys)

**Comment by SpecialActivities** [comment]
ah unfortunately that was the method i tried, sadly it didn't work. Is there a workaround beyond this? i've tried hard but even that method didn't resolve the issue ><

**Comment by CrazyRedMachine** [comment]
oh damn, did they change something again (it used to work with early win10 versions, then there was that update with the workaround, then now this... ). I'll try to have a new look at this project soon, currently busy with other works...

sorry about your keyboard, in the mean time it should still work with ubuntu or win7 :/

I'll leave this issue open and comment on it when I find a solution so you'll get notified.

**Comment by SpecialActivities** [comment]
> oh damn, did they change something again (it used to work with early win10 versions, then there was that update with the workaround, then now this... ). I'll try to have a new look at this project soon, currently busy with other works...
> 
> sorry about your keyboard, in the mean time it should still work with ubuntu or win7 :/
> 
> I'll leave this issue open and comment on it when I find a solution so you'll get notified.

Would it be possible to create a working driver set for the 2nd HID with Built-in Midi Detection/Input/Output? Wondering if I can help at all. I certainly think this is a native issue with Windows 10 updates and driver updates too.

**Comment by anonona91** [comment]
Hey CRM, any updates yet bud?
Thanks.

**Comment by CrazyRedMachine** [comment]
Hi,
I'm currently working on other stuff so this project has very low priority at the moment, sorry.

I'll be sure to leave a note here when it's fixed :)

**Comment by anonona91** [comment]
> Hi, I'm currently working on other stuff so this project has very low priority at the moment, sorry.
> 
> I'll be sure to leave a note here when it's fixed :)

Do you accept PayPal donation's CRM?
This is greatly appreciated btw!

**Comment by CrazyRedMachine** [comment]
thanks for asking, yes I do, the link is up on my other active repos :)

**Comment by anonona91** [comment]
> thanks for asking, yes I do, the link is up on my other active repos :)

I'll make a donation when I get home. All the best bud.

**Comment by CrazyRedMachine** [comment]
thanks, much appreciated :) i'll take the time to test another solution i had in mind for these keyboards very soon

**Comment by anonona91** [comment]
> thanks, much appreciated :) i'll take the time to test another solution i had in mind for these keyboards very soon

10 Euros is on your way mate. Best of luck.

**Comment by CrazyRedMachine** [comment]
thank you very much !

**Comment by nanocyde** [comment]
Hey, I'm having the same issue on Win10 x64, where either the note input works or the PC keyboard does depending on the driver. I would like to donate if you have the time to look at this project.

**Comment by nanocyde** [comment]
Can you at least let me know if this is feasible? Thanks

**Comment by CrazyRedMachine** [comment]
hi,
sorry about the delay, but everything has already been said in this thread (see  https://github.com/CrazyRedMachine/Prodikeys64/issues/3#issuecomment-1022714984 ) and i've got no additional information for now as I'm still working on that other project I mentioned.

I'm not sure if it is feasible to solve it only via software i have to try a couple things, worst case I'll make a hardware solution

**Comment by lyh1win** [comment]
install the driver for both interfaces make midi key work for windows 10, But the fn_state dont change when press the fn key. Is it a software function?

**Comment by upulkkis** [comment]
Got the current version working with the latest win10. Just follow the instructions of the author (replacing the driver only for interface 1). After that unplug and then reattach your prodikeys device. Before starting prodikeys64.exe, replace the libusb-1.0.dll with the libusb-1.0.26-rc1-binaries (the x64 visual studio version) from libusb github releases page. After that the both, keyboard and midi keys, are working as descibed in author's description. Thank you @CrazyRedMachine so much for this driver, I've been waiting for a decade to be able to use it again!

**Comment by CrazyRedMachine** [comment]
ooh this is awesome news!! i didn't know libusb had updated and fixed this... thank you very much for this info :)) I will try it this week, and prepare a new release with the correct dll builtin if it works for me too!

yea it's a shame they never released another prodikeys, they were such cool products

**Comment by djolejov** [comment]
> Got the current version working with the latest win10. Just follow the instructions of the author (replacing the driver only for interface 1). After that unplug and then reattach your prodikeys device. Before starting prodikeys64.exe, replace the libusb-1.0.dll with the libusb-1.0.26-rc1-binaries (the x64 visual studio version) from libusb github releases page. After that the both, keyboard and midi keys, are working as descibed in author's description. Thank you @CrazyRedMachine so much for this driver, I've been waiting for a decade to be able to use it again!

This works for me! Thank you very much!

**Comment by CrazyRedMachine** [comment]
@VolumeInfinite @anonona91 @nanocyde 
finally got some time to retrieve the keyboard and properly test libusb update here, v1.1 has been released, sorry about the delay

@lyh1win make sure you're pressing the piano key on very top left of keyboard, not fn key at top left of numpad

**Comment by Estabanx** [comment]
I tested it on both operating systems and it works on both. WIN10 and WIN11 . congratulations for your work. 
Suggestion: Prodikeys64 would be better if it had the option to auto start when windows starts. I always start it manually.

thank you, @CrazyRedMachine

**Comment by CrazyRedMachine** [comment]
thanks, glad to know it works on win11 too :D

to have start automatically you can add a shortcut to prodikeys64 in the startup folder (win+r then type "shell:startup" and enter, it will open a folder, drag&drop prodikeys64 **with right click** then "create shortcut here")
